packages/jest-circus/src/run.ts

```diff
@@ -48,9 +48,14 @@ const _runTestsForDescribeBlock = async (describeBlock: DescribeBlock) => {
   const deferredRetryTests = [];
 
   for (const test of describeBlock.tests) {
+    const isErrorsBeforeTestRun = test.errors.length > 0;
```

Comment:
Nit: maybe `hasErrorsBeforeTestRun`?

Reply:
Good call. I pushed another commit for that just now.